lib/manager/bundler/readme.md

```diff
@@ -4,39 +4,29 @@
 
 #### Name of package manager
 
-[Bundler](https://bundler.io/)
-
----
+Bundler
 
 #### What language does this support?
 
 Ruby
 
----
-
 #### Does that language have other (competing?) package managers?
 
-No mainstream
+No
 
 ## Package File Detection
 
 #### What type of package files and names does it use?
 
-"Gemfiles", usually named `Gemfile`.
-
----
+Package - gem. Name - Gemfile or gemspec
 
 #### What [fileMatch](https://renovatebot.com/docs/configuration-options/#filematch) pattern(s) should be used?
 
-`["(^|/)Gemfile$]`
-
----
+`/Gemfile|.gemspec/`
```

Comment:
Do we want to find any file with `Gemfile` or `.gemspec` in its name *or* path, or just in its name, or do we want an exact match of the filename?

Reply:
I think for `Gemfile` we want to find files with exec matches and for `.gemspec` any file that matches pattern, but I'm not really sure that we should care about `.gemspec` files at first implementation 

https://bundler.io/v1.17/rubygems.html#using-bundler-with-rubygem-gemspecs